bnb_yaz/new_bnb_downloader.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# A tool for the automation of MARC downloads from the BNB.
+# This version reads BNB IDs from a PDF file and downloads records from the BNB server.
+# Author: Steven Lolong (steven.lolong@uni-tuebingen.de)
+# First created: November 2025
+# Usage: new_bnb_downloader.py <email> <working_directory>
+# Requires 'pdftotext' utility to parse PDF files.
+# The working directory should have the following structure:
+# The 'input' directory should contain the PDF files with BNB IDs.
+# The 'loaded' directory will store processed PDF files and successfully uploaded MARC files.
+# The 'marc' directory will store downloaded MARC files.
+# The 'logs' directory will store log files.
+
+
+import pexpect
+import re
+import os
+import sys
+import sqlite3
+import datetime
+# import bsz_util
+
+# Global variables
+db_connection : sqlite3.Connection
+bnb_yaz_client : pexpect.spawn
+pdf_file_name : str
+marc_output_file : str
+tmp_parse_file : str
+working_directory : str
+email_recipient : str
+total_new_extracted_ids : int = 0
+total_found_ids : int = 0
+total_not_found_ids : int = 0
+total_existing_ids : int = 0
+pdf_files : list = []
+new_bnb_info : dict = {}
+
+DATABASE_NAME = "/usr/local/var/lib/tuelib/bnb_downloads.sq3"
```

Comment:
The `/usr/local/var/lib/tuelib` directory is referenced on multiple occasions in this script, as well as others. I think we should put this directory path into a constant.